presentation/src/main/java/com/mashup/gabbangzip/sharedalbum/presentation/ui/common/TopBar.kt

```diff
@@ -56,10 +60,12 @@ fun TopBar(
             Text(
                 modifier = Modifier.weight(1.0f),
                 text = if (titleAlign == TopBarTitleAlign.LEFT) titleText else "",
+                style = titleStyle,
             )
             if (rightIcon1?.resId != null) {
                 Image(
                     modifier = Modifier
+                        .clickable { rightIcon1.iconClickListener.invoke() }
```

Comment:
이게 clickable이 기본 리플이 뚝뚝 끊기는 것처럼 안이쁘다라고 하더라고..? 그래서 아래 코드 처럼 리플을 저렇게 적용해주더라고..! 한번 제안해 봤어~ㅎ

```suggestion
.clickable(
                interactionSource = remember { MutableInteractionSource() },
                indication = rememberRipple(),
              )
```

Reply:
오호 뭔가 미묘하게 다르긴 하네..? 형이 올려준게 조금 더 부드러운 느낌이 있다.. 왜지..? 아무튼 반영완료! https://github.com/mash-up-kr/gabbangzip-Android/pull/31/commits/ca2ae99025309f0436c0fbce7cb0161c2f63af9d

근데 나 이거 왜 그런지 궁금한데 혹시 막 참고했던 레퍼런스같은거 있어..?